src/StackAdmin/AzureStack/AzureStack.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule             = "AzureStack.psm1"
 
     # Version number of this module.
-    ModuleVersion          = '1.3.0'
+    ModuleVersion          = '1.4.0'
```

Comment:
We need matching changes to the release notes, correct?  Same for the individual admin module changes.

Reply:
added release notes